device/cuda/src/finding/finding_algorithm.cu

```diff
@@ -21,7 +21,9 @@
 
 // detray include(s).
 #include "detray/core/detector.hpp"
-#include "detray/detectors/detector_metadata.hpp"
+#include "detray/core/detector_metadata.hpp"
```

Comment:
```suggestion
```
Unless you want to switch to the full metadata now, in which case please remove the telescope and toy metadata instead and use
```
using host_detector_type = detector<>;
using device_detector_type = detector<default_metadata, , covfie::field_view,
                     detray::device_container_types>;
```

Reply:
I will do that at some point. But I am trying to keep the changes as small as possible for each PR.